apps/sms/style/sms.css

```diff
@@ -962,14 +962,20 @@ section[role="region"][class*='-information'] #messages-compose-form {
 }
 
 /* Phone details label styles */
-.phone-type,
+.phone-type {
+  display: -moz-inline-box;
+  max-width: 60px;
```

Comment:
can you please experiment with flex box for this "subheader" part? You can do this in another branch and pull request so that we can compare.